skins/moono-lexicon/icons/italic.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+	<path class="lexicon-icon-outline italic-i-stem" d="M165.23 512h92.562l60.343-342.235h-92.526z"></path>
+	<path class="lexicon-icon-outline italic-i-dot" d="M330.423 16.713c-10.935-11.154-24.503-16.713-40.704-16.713-15.762 0-29.221 5.559-40.338 16.713-11.154 11.118-16.713 24.832-16.713 41.033 0 15.323 5.559 28.489 16.713 39.643 11.118 11.118 24.576 16.713 40.338 16.713 16.201 0 29.769-5.559 40.704-16.713 10.862-11.154 16.347-24.357 16.347-39.643 0-16.238-5.486-29.915-16.347-41.033z"></path>
+</svg>
```

Comment:
Sucks that we'll have to keep these in sync manually, but at least you're making a nice script to automate the build

Reply:
Why not add a dependency on `@clayui/css`? Don't we publish these in `images/icons`?

Reply:
Good idea.

```
└── @clayui
    └── css
        ├── CHANGELOG.md
        ├── LICENSE.txt
        ├── README.md
        ├── index.js
        ├── lib
        │   ├── css
        │   │   ├── atlas.css
        │   │   ├── atlas.css.map
        │   │   ├── base.css
        │   │   ├── base.css.map
        │   │   ├── bootstrap.css
        │   │   └── bootstrap.css.map
        │   ├── images
        │   │   └── icons
        │   │       ├── add-cell.svg
        │   │       ├── add-column.svg
        │   │       ├── add-role.svg
        │   │       ├── add-row.svg
        │   │       ├── adjust.svg
        │   │       ├── ...
 ```

Reply:
Yep, I was thinking about adding it as a dependency. Wondering if we can take advantage of it and use scss variables so we don't need to hardcode'em...

Reply:
Anyway, @jbalsas @wincent Clay has tons of icons we don't need here, plus not all the names are the same CKEditor needs, so to do this we'll need to create some mapping file in the skin to copy and rename only the icons we need.

Reply:
That sounds exactly like the type of work for a build script 😂 

I wouldn't mind hardcoding them temporarily and see where that lead us